src/index.html

```diff
@@ -5,9 +5,45 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./styles/index.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__img"
+        src="../src/images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock"
+      >
+      <h2 class="card__name">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h2>
```

Comment:
I would not use `hx` tags in such a task.

Let's make it clear - in the context of your task it is ok, as you have a single product card per your entire page. But imagine a real-world store with multiple product cards on the page - that will lead to dozens `h2`s and even worse dozens of _the same_ `h3`'s .  The latter would almost definitely hurt SEO and page readability for screen readers.

Unfortunately, I am not a SEO specialist to give you a deep explanation, but most probably you should avoid using `hx` tags in such blocks

Probably this article will give you more idea:
https://semji.com/blog/why-use-h1-h2-h3-tags-for-your-seo